glyphs-reader/src/font.rs

```diff
@@ -245,11 +260,23 @@ pub struct Anchor {
 #[derive(Debug, Clone, FromPlist, PartialEq, Eq, Hash)]
 pub struct FontMaster {
     pub id: String,
+    pub ascender: Option<OrderedFloat<f64>>,
+    pub descender: Option<OrderedFloat<f64>>,
+    pub baseline: Option<OrderedFloat<f64>>,
+    pub cap_height: Option<OrderedFloat<f64>>,
+    pub x_height: Option<OrderedFloat<f64>>,
     pub axes_values: Option<Vec<OrderedFloat<f64>>>,
+    pub metric_values: Option<Vec<RawMetricValue>>,
```

Comment:
this again looks a bit weird to me, because metrics_values technically only exists in v3 sources, whereas ascender/descender/etc. only exist in v2 sources, but you're using the same FontMaster struct for either v2 or v3...

Reply:
I was trying to avoid having RawFontMaster and FontMaster, probably I should just bite the bullet and do it.

Reply:
We parse both to the same structures so I think that's only avoidable by some really nasty hand-parsing of plists. Per prior comments, I'll make that just on RawFontMaster so you don't "see" it if you interact with only Font.